annotations/tests/applicators.json

```diff
@@ -0,0 +1,409 @@
+{
+  "$schema": "../test-suite.schema.json",
```

Comment:
Is this a conflict of message?

Ref: https://github.com/orgs/json-schema-org/discussions/473

Reply:
I don't think so. I don't think we decided that it's wrong to use `$schema` this way. We just decided that we don't want to standardize using it that way. The point is to make use of the tooling that's in common use. I think it's fine.